specs/resources/exchanges/listCreateExchangeForAccount.yml

```diff
@@ -0,0 +1,195 @@
+          schema:
+            type: object
+            properties:
+              _links:
+                type: object
+                properties:
+                  self:
+                    type: object
+                    properties:
+                      href:
+                        type: string
+                        example: https://api-sandbox.dwolla.com/exchanges/fcd15e5f-8d13-4570-a9b7-7fb49e55941d
+                      type:
+                        type: string
+                        example: application/vnd.dwolla.v1.hal+json
+                      resource-type:
+                        type: string
+                        example: exchange
+                  exchange-partners:
+                    type: object
+                    properties:
+                      href:
+                        type: string
+                        example: https://api-sandbox.dwolla.com/exchange-partners/9b55a4b3-34ae-4607-b2d1-622f1eed77f9
+                      type:
+                        type: string
+                        example: application/vnd.dwolla.v1.hal+json
+                      resource-type:
+                        type: string
+                        example: exchange-partner
+                  id:
+                    type: string
+                    example: 'fcd15e5f-8d13-4570-a9b7-7fb49e55941d'
+                  status:
+                    type: string
+                    example: active
+                  created:
+                    type: string
+                    example: '2022-10-21T21:41:03.283Z'
+post:
+  tags:
+    - exchange
+  summary: Create exchange for an account
+  description: Create exchange for an account
+  operationId: createExchangeForAccount
+  requestBody:
+    content:
+      application/json:
+        schema:
+          oneOf:
+            - $ref: '#/components/schemas/FinicityJSONObjectSchema'
+            - $ref: '#/components/schemas/MXObjectSchema'
+  responses:
+    '201':
+      description: Created
+      headers:
+        location:
+          type: string
+          example: https://api-sandbox.dwolla.com/exchanges/fcd15e5f-8d13-4570-a9b7-7fb49e55941d
+      content:
+        application/vnd.dwolla.v1.hal+json:
+          schema:
+            type: object
+    '400':
+      description: Bad Request
+      content:
+        application/vnd.dwolla.v1.hal+json:
+          schema:
+            oneOf:
+              - $ref: '#/components/schemas/InvalidExchangeTokenSchema'
+              - $ref: '#/components/schemas/InvalidExchangeSchema'
+    '401':
+      description: Invalid Scope
+      content:
+        application/vnd.dwolla.v1.hal+json:
+          schema:
+            type: object
+            properties:
+              code:
+                type: string
+                example: InvalidExchange
+              message:
+                type: string
+                example: The exchange is no longer active.
+components:
+  schemas:
+    FinicityJSONObjectSchema:
```

Comment:
Suggested edit on naming convention. Should we call this `FinicityCreateExchangeSchema`?